packages/cfd/src/Containers/mt5-trade-modal.tsx

```diff
@@ -78,7 +78,6 @@ const MT5TradeModal = ({
                     toggleModal={toggleModal}
                     should_header_stick_body={false}
                     width='600px'
-                    height={platform === 'mt5' ? '720px' : '500px'}
```

Comment:
Since the DxTrade modal is smaller, wont this make the modal bigger ?

Reply:
Height is `auto` so it will adjust to the content of the modal.